pylintrc

```diff
@@ -89,10 +89,15 @@ confidence=
 disable=
     fixme,
     locally-disabled,
+    no-self-use,
```

Comment:
Do you think it would be possible to apply `@staticmethod` to those methods instead of ignoring every occurrence?

Reply:
Yes it would. I thought best to just ignore them all because:
* The dozens of exceptions in the codebase show the developers do not generally value this
* I don't see a value in this stylistic convention, i.e. what exactly is the problem with a self that's not used?
* The edx-lint has [no-self-use](https://github.com/edx/edx-lint/blob/master/edx_lint/files/pylintrc#L303) disabled which probably has an influence in the fact that developers do not value this stylistic convention in the Open edX ecosystem

That bein said, if you think it must be enforced I will remove this exception and fix all issue.

Reply:
No need to, I'm convinced 😛